catroid/src/main/java/org/catrobat/catroid/ProjectManager.java

```diff
@@ -77,13 +78,34 @@
 	private Scene startScene;
 	private Sprite currentSprite;
 
-	private ProjectManager() {
+	private Context applicationContext;
+
+	public ProjectManager(Context applicationContext) {
+		if (instance != null) {
+			throw new RuntimeException("ProjectManager should be instantiated only once");
+		}
+		this.applicationContext = applicationContext;
+		instance = this;
 	}
 
+	/**
+	 * Replaced with dependency injection
+	 *
+	 * @deprecated use dependency injection with Dagger instead.
+	 */
+	@Deprecated
 	public static ProjectManager getInstance() {
 		return INSTANCE;
```

Comment:
```
[2019-10-01T07:16:01.665Z] /home/catroid/jenkins_slave/workspace/Catroid_PR-3353/catroid/src/main/java/org/catrobat/catroid/ProjectManager.java:98: error: cannot find symbol
[2019-10-01T07:16:01.665Z] 		return INSTANCE;
[2019-10-01T07:16:01.665Z] 		       ^
[2019-10-01T07:16:01.665Z]   symbol:   variable INSTANCE
[2019-10-01T07:16:01.665Z]   location: class ProjectManager
```

Reply:
You should never rename a variable manually 😉
Resolved